pkg/api.go

```diff
@@ -85,6 +87,97 @@ type CredentialWriter interface {
 	Add(vc *verifiable.Credential) error
 }
 
+// Crypto defines various crypto operations that may be used with wallet-sdk APIs.
+type Crypto interface {
+	// Encrypt will encrypt msg and aad using a matching AEAD primitive in kh key handle of a public key
+	// returns:
+	// 		cipherText in []byte
+	//		nonce in []byte
+	//		error in case of errors during encryption
+	Encrypt(msg, aad []byte, kh interface{}) ([]byte, []byte, error)
+	// Decrypt will decrypt cipher with aad and given nonce using a matching AEAD primitive in kh key handle of a
+	// private key
+	// returns:
+	//		plainText in []byte
+	//		error in case of errors
+	Decrypt(cipher, aad, nonce []byte, kh interface{}) ([]byte, error)
```

Comment:
@sudeshrshetty @DRK3 Do we need Encrypt/Decrypt/DeriveProof/Blind for OpenID flows ? We can start with minimum and then add functions.

Reply:
Just to confirm: do we need all of the methods _except_ for those specific 4 ones you listed? Or is there a smaller subset?

Reply:
For now, just keep Sign/Verify.

Reply:
Done